packages/server/src/queue/queue.entity.ts

```diff
@@ -82,11 +89,18 @@ export class QueueModel extends BaseEntity {
   queueSize: number;
 
   async addQueueSize(): Promise<void> {
-    this.queueSize = await QuestionModel.waitingInQueue(this.id).getCount();
+    this.queueSize = await QuestionModel.inQueueWithStatus(this.id, [
+      ...StatusInQueue,
+      OpenQuestionStatus.Helping,
+    ]).getCount();
   }
```

Comment:
makes the queue.queueSize include students that are currently being helped (since students will see 0 students in a queue and probably think the TA is free)